SasquatchMac/SasquatchMac/ViewControllers/TransmissionViewController.swift

```diff
@@ -443,7 +443,7 @@ class TransmissionViewController: NSViewController, NSTableViewDataSource, NSTab
     collectDeviceIdStates[selectedTarget!] = true
   }
 
-  override func controlTextDidChange(_ obj: Notification) {
+  func controlTextDidChange(_ obj: Notification) {
```

Comment:
Does it work now?

Reply:
There is something wrong here. I used the wrong function. I fix it immediately.

Reply:
it does worked.

Reply:
it does worked.